modules/persistent-storage-csi-vsphere-disable-storage-consequences.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-disable-storage-consequences_{context}"]
+= Consequences of disabling and enabling storage on vSphere
+
+The consequences of disabling and enabling storage on vSphere are described in the following table.
+
+.Consequences of disabling/enabling storage on vSphere
+|===
+|Disabling | Enabling
+
+a| * vSphere CSI Driver Operator un-installs the CSI driver.
+
+* Storage container orchestration (CO) should be healthy.
+
+* vSphere-problem-detector continues running, but does not emit alerts or events, and checks less frequently (once per 24 hours).
+
```

Comment:
worth mentioning that the operator itself it not removed and keeps running?
@gnufied @mpatlasov

Reply:
I think that is implementation detail. We can leave that part out. my 2 cents.

Reply:
I agree, we can leave that part out. Users will know that operator itself is not removed reading these lines: https://github.com/openshift/openshift-docs/pull/80535/commits/08b53b2d08d7eb9095c9ec2fc625d47f7dccd195#diff-04fbf73bdcb4f88b4df88f78ecc63147576de5e001479398f252bf29699cc9f2R42-R43

Reply:
ok works for me

Reply:
Seems like we've decided not to include this either.